oxigen_stock_barcodes_gs1/wizard/stock_barcodes_read.py

```
@@ -16,11 +16,14 @@ def process_lot(self, barcode_decoded):
         res = super().process_lot(barcode_decoded)
         if self.product_id.tracking == "serial":
             lot_barcode = barcode_decoded.get("21", False)
+            operation = getattr(self, "picking_id", False) or getattr(
+                self, "inventory_id", False
+            )
```

What happens if neither of the operations exist? Should we return an error?